deps/db/src/logseq/db/frontend/property.cljs

```diff
@@ -38,6 +38,90 @@
                                    :cardinality :many
                                    :public? true
                                    :classes #{:logseq.class/Root}}}
+   :kv/value             {:title "KV value"
+                          :attribute :kv/value
+                          :schema {:type :any
+                                   :public? false
+                                   :hide? true}}
+   :block/type           {:title "Node type"
+                          :attribute :block/type
+                          :schema {:type :string
+                                   :public? false
+                                   :hide? true}}
+   :block/schema         {:title "Node schema"
+                          :attribute :block/schema
+                          :schema {:type :map
+                                   :public? false
+                                   :hide? true}}
+   :block/parent         {:title "Node parent"
+                          :attribute :block/parent
+                          :schema {:type :entity
+                                   :public? false
+                                   :hide? true}}
+   :block/order          {:title "Node order"
```

Comment:
:block/order is no longer a queryable property.

Reply:
> I haven't looked at the rest of these attributes but there is probably a lot more to be said about them...

I looked at the rest of these and disabled more that are implementation details that could change. I would've disabled :block/type as it's an also implementation detail but left it alone since you wanted to make it queryable with https://github.com/logseq/logseq/commit/94773db6f3e17a3e4f0ca42d8e9dcf0193d9c7f5